components/track/TrackMenu.vue

```diff
@@ -62,7 +64,20 @@ const dropdownMenuItemsForTrack = (track: TrackModel) => {
   items.push({
     icon: "icon.person",
     text: t("track.dropdown.go-to-contributors"),
-    link: { name: "browse" }, // TODO: change link
+    clickFunction: () => {
+      if (track.contributors && track.contributors?.length > 1) {
+        showContributorsList.value = true;
+      } else if (
+        track.contributors &&
+        track.contributors.length === 1 &&
+        track.contributors[0]?.id
+      ) {
+        navigateTo(`/playlist/contributor/${track.contributors[0].id}`);
```

Comment:
Please use an object-structure for navigation instead. This will avoid dead-links as we by type-safety can ensure those links actually exist in the router.